lib/brain.js

```diff
@@ -1546,6 +1546,96 @@ Brain.prototype.twoWayMode = function twoWayMode () {
   return this.trader.twoWayMode
 }
 
+/**
+ * Check if the customer is suspended
+ *
+ * That happens if the customer has reached
+ * one of the enabled compliance tier thresholds
+ *
+ * @name isSuspended
+ * @function
+ *
+ * @param {object} customer Acting customer
+ * @param {date} now Current date
+ * @returns {bool} Whether customer is suspended or not
+ */
+Brain.prototype.isSuspended = function isSuspended (customer, now) {
```

Comment:
I don't see a reason why now is a param, can't we just create a new Date within this function?